cmd/thanos/query.go

```diff
@@ -74,6 +74,9 @@ func registerQuery(m map[string]setupFunc, app *kingpin.Application) {
 	selectorLabels := cmd.Flag("selector-label", "Query selector labels that will be exposed in info endpoint (repeated).").
 		PlaceHolder("<name>=\"<value>\"").Strings()
 
+	requiredMatcher := cmd.Flag("required-matcher", "Matchers that will be required in order to processes any request to `Series` (or `Select`  via query HTTP). If request matches will not have matchers required, query will return error. (repeated).").
```

Comment:
`.Hidden()`

```suggestion
	requiredMatcher := cmd.Flag("required-matcher", "Experimental: Matchers that will be required in order to processes any request to `Series` (or `Select`  via query HTTP). If request matches will not have matchers required, query will return error. (repeated).").
```

+ Let's make it matcher (: